### ISSUE TITLE
Turning on minifier causes bug to go away (on DebertaForMaskedLM)

### ISSUE THREAD

**Issue by ezyang** [pytorch/pytorch#93592]
### 🐛 Describe the bug

1. Check out and build pytorch at e4a8661ab84022c1bff622c6d2f6e679180b1df5 (Oct 29, before breaking commit was reverted)
2. Run `python benchmarks/dynamo/huggingface.py  --accuracy --backend inductor --training --only DebertaForMaskedLM`, it fails
3. Run `TORCHDYNAMO_REPRO_AFTER=dynamo python benchmarks/dynamo/huggingface.py  --accuracy --backend inductor --training --only DebertaForMaskedLM`, it passes

### Error logs

https://www.internalfb.com/intern/paste/P543371878/

### Minified repro

minifier did not work
